README.md

```diff
@@ -84,6 +84,7 @@ AutoGluon also provides state-of-the-art tools for neural hyperparameter and arc
 - [General introduction into HNAS](https://www.youtube.com/watch?v=pB1LmZWK_N8&feature=youtu.be)
 - [Introduction into HNAS with AutoGluon](https://www.youtube.com/watch?v=GJVwUyVWZas)
 - [Example notebook](https://github.com/zhanghang1989/HPO2NAS-Tutorial-CVPR-ECCV2020/blob/master/mlp.ipynb)
+- [Example scripts that show how we can run AutoGluon on different neural network benchmarks](https://github.com/awslabs/autogluon/tree/master/examples/hnas/)
```

Comment:
I think this could be more clearly written as something like:

"Example script for efficient Hyperband-style tuning of PyTorch neural network hyperparameters"

Reply:
ok I changed it to "Example scripts for efficient multi-fidelity HNAS of PyTorch neural network model" (more scripts to follow)